### PR TITLE
[itest] Applied new source code formatter tool

### PR DESCRIPTION
**Pull request by cweitkamp**
- Applied new source code formatter tool

During the application the logs showed the following warning - Does anybody know what they are meaning?
```
[INFO] ------< org.openhab.core.itests:org.openhab.core.reactor.itests >-------
[INFO] Building openHAB Core :: Integration Tests 2.5.0-SNAPSHOT         [1/30]
[INFO]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] --- spotless-maven-plugin:1.24.3:apply (default-cli) @ org.openhab.core.reactor.itests ---
[WARNING] The POM for com.sun.xml.bind:jaxb-core:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
[WARNING] The POM for com.sun.xml.bind:jaxb-impl:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
```

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTest.bundle/ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -46,7 +46,8 @@
 		<item-type>ColorItem</item-type>
 		<label>HUE Lamp Color</label>
 		<description>The color channel allows to control the color of the hue
```
Comment (cweitkamp) [inline itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTest.bundle/ESH-INF/thing/thing-types.xml]: This part looks weired.

File: itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTestNoAuthor.bundle/ESH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -46,7 +46,8 @@
 		<item-type>ColorItem</item-type>
 		<label>HUE Lamp Color</label>
 		<description>The color channel allows to control the color of the hue
```
Comment (cweitkamp) [inline itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTestNoAuthor.bundle/ESH-INF/thing/thing-types.xml]: This part looks weired.

File: itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/yahooweather.bundle/ESH-INF/binding/binding.xml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+<binding:binding id="yahooweather" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
 
-    <name>YahooWeather Binding</name>
-    <description>The Yahoo Weather Binding requests the Yahoo Weather Service
-		to show the current temperature, humidity and pressure.</description>
-    <author>Kai Kreuzer</author>
+	<name>YahooWeather Binding</name>
+	<description>The Yahoo Weather Binding requests the Yahoo Weather Service
```
Comment (cweitkamp) [inline itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/yahooweather.bundle/ESH-INF/binding/binding.xml]: This part looks weired.

File: itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTest.bundle/ESH-INF/thing/thing-types.xml (hunk 1 above)
Comment (seime) [reply itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTest.bundle/ESH-INF/thing/thing-types.xml]: I agree. If you try to reformat it inside Eclipse you get the same result. It is probably related to previous manual formatting - if you remove the previous line breaks and then reformat the output is as it is supposed to be.

Comment (cweitkamp) [reply itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTest.bundle/ESH-INF/thing/thing-types.xml]: I changed it by hand.

**Comment by kaikreuzer** [comment]
> During the application the logs showed the following warning - Does anybody know what they are meaning?

Maybe it is due to a missing `JAVA_HOME`? See https://stackoverflow.com/questions/26393332/getting-the-pom-for-name-is-invalid-transitive-dependencies-if-any-will-no

**Pull request merged**